src/forder.c

```diff
@@ -1222,8 +1227,14 @@ void radix_r(const int from, const int to, const int radix) {
     } else {
       // all groups are <=65535 and radix_r() will handle each one single-threaded. Therefore, this time
       // it does make sense to start a parallel team and there will be no nestedness here either.
+
+      int threads_to_run = MIN(nth, getDTthreads(ngrp, false));  // the parallel regions below might reach code
```

Comment:
Calling `getDTthreads()` again here still leaves the same door open that this value of throttle needs to be the same as the value used to allocate on line 718. We can use `nth` on the `#pragma` line, avoid the new variable, avoid the extra call to `getDTthreads()` and reduce from 3 places to 2 places. Will do...

Reply:
I don't think calling it again here with the `MIN` limiting it to the value of `nth` is necessarily the same problem. But thinking about it again and seeing the last commits you added, I'd say it was redundant, as `getDTthreads` w/ `throttle=false`, basically only limits to the number of configured threads, which is already done w/ `nth`. So if configured threads are the limiting factor, this already resembled in `nth` so there is no need to apply it again to `ngrp` as it can then only play a role here if `ngrp < threads`, but then also `ngrp < nth` would be true. So doing like you did with `MIN(nth, grp)` is more efficient as it saves the function call.